src/index.html

```diff
@@ -22,6 +22,402 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="https://apple.com"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://samsung.com"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://sv.wikipedia.org/wiki/Smartmobil"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Laptop"
+              data-qa="nav-hover"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Gadget"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Tablet_computer"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Photograph"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__img">
+          <img
+            class="card__item"
+            src="./images/imac.jpeg"
+            alt="imac"
```

Comment:
alt attribute should describe the image content
<img width="512" alt="image" src="https://github.com/user-attachments/assets/da7fdae4-70d0-47e5-a3d6-659a6a38c5e7">